experimental/vlan-networks.md

```diff
@@ -119,7 +119,7 @@ For the driver to add/delete the vlan sub-interfaces the format needs to be `int
 
 The option to use either existing parent vlan sub-interfaces or let Docker manage them enables the user to either completely manage the Linux interfaces and networking or let Docker create and delete the Vlan parent sub-interfaces (netlink `ip link`) with no effort from the user.
 
-For example: `eth0.10` to denote a sub-interface of `eth0` tagged with vlan id `10`. The equivalent `ip link` command would be `ip link add link eth0 name eth0.10 type vlan id 10`.
+For example: for `eth0.10` to denote a sub-interface of `eth0` tagged with the vlan id of `10`. The equivalent `ip link` command would be `ip link add link eth0 name eth0.10 type vlan id 10`.
```

Comment:
hm yeah `the` and `of` are not really needed; perhaps

    use `eth0.10` to denote ...

instead of

    for eth0.10` to denote